### ISSUE TITLE
Filter output

### ISSUE THREAD

**Issue by TrySound** [kevva/download#59]
In `run(err, files)` files contains directories too. Can you filter it?

**Comment by TrySound** [comment]
I use extract:true and it gives broken file.stat
```js
{ dev: undefined,
  mode: undefined,
  nlink: undefined,
  uid: undefined,
  gid: undefined,
  rdev: undefined,
  blksize: undefined,
  ino: undefined,
  size: undefined,
  blocks: undefined,
  atime: Invalid Date,
  mtime: Tue Mar 17 2015 20:43:08 GMT+0300 (MSK),
  ctime: Invalid Date,
  birthtime: Invalid Date }
```

**Comment by shinnn** [comment]
Which file type of archives are you trying to extract?

> Can you filter it?

No. Filter them by yourself. Some archives contain meaningful empty directories as needed.

**Comment by shinnn** [comment]
OK, broken file.stat is a separate issue. Please reopen it on the correct [decompress-plugin](https://github.com/search?q=user%3Akevva+decompress-) repo.
